### DISCUSSION TITLE
Branch snippet inclusion

### DISCUSSION THREAD

**Discussion by ryanbr** [uBlockOrigin/uAssets#14785] [General]
`@@||cdn.branch.io/branch-latest.min.js$script` is currently used.

Mozilla is using: `https://github.com/mozilla/gecko-dev/blob/master/browser/extensions/webcompat/shims/branch.js` 

Related ticket: https://bugzilla.mozilla.org/show_bug.cgi?id=1716220  Problem site: https://watch.tatasky.com/

Possible to include a branch snippet rather than having this global exception?

**Comment by ghajini** [comment]
@gwarser @gorhill 
many
https://github.com/mozilla/gecko-dev/tree/master/browser/extensions/webcompat/shims
